elasticsearch/src/main/java/org/hibernate/search/elasticsearch/schema/impl/model/PropertyMapping.java

```diff
@@ -119,6 +126,25 @@ public void setNullValue(JsonPrimitive nullValue) {
 		this.nullValue = nullValue;
 	}
 
+	public Map<String, PropertyMapping> getFields() {
+		return fields;
+	}
+
+	private Map<String, PropertyMapping> getInitializedFields() {
```

Comment:
We use Gson, not Jackson. Gson seems only serializes fields, and not methods (unless we use custom serializers).

The reflective type adapter (the default) seems to really only consider fields: https://github.com/google/gson/blob/gson-parent-2.4/gson/src/main/java/com/google/gson/internal/bind/ReflectiveTypeAdapterFactory.java#L147

And anyway, the `Expose` annotation (used to ignore data when we cannot mark fields as "transient") can only be applied to fields, not to methods:

```
@Target(ElementType.FIELD)
public @interface Expose {
```

Also, *if* the result of this method was being serialized, a good load of tests would be failing, because "initializedFields" would not be understood by Elasticsearch.